packages/@o3r/design/cli/generate-css-from-design-token.cli.cts

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+import { isAbsolute, normalize, resolve } from 'node:path';
+import { existsSync } from 'node:fs';
+import { parseDesignTokenFile, renderDesignTokens } from '@o3r/design';
+import type { DesignTokenRendererOptions, DesignTokenVariableSet } from '@o3r/design';
+import * as minimist from 'minimist';
+
+const args = minimist(process.argv.splice(2));
+
+void (async () => {
+  const renderDesignTokenOptions: DesignTokenRendererOptions = {};
+
+  const output = args.o || args.output;
+  if (output) {
+    renderDesignTokenOptions.determineFileToUpdate = () => resolve(process.cwd(), output);
+  }
+
+  const tokens = (await Promise.all(
+    args._
+      .map((file) => isAbsolute(file) ? normalize(file) : resolve(process.cwd(), file))
+      .filter((file) => {
+        const res = existsSync(file);
+        if (!res) {
+          throw new Error(`The file ${file} does not exist, the process will stop`);
```

Comment:
```suggestion
          throw new O3rCliError(`The file ${file} does not exist, the process will stop`);
```

Reply:
Not sure it is a good idea to have it in this CLI.

Currently this CLI can be used just with `npx @o3r/design` without additional depdendency.
If I add `O3rCliError`, I will need to add a dep to `@o3r/schematics` (and it's peer deps) as well.